inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
+getPdbIds <- function(targetSeq, startPos = 1 , endPos = nchar(targetSeq), convert = TRUE, silent = TRUE){
```

Comment:
You can, but I realized that, if you look a couple lines down, the endPos is automatically changed to the length of the amino acid sequence (the default is set to the nuc seq length)